### ISSUE TITLE
Fix for bypass req?

### ISSUE THREAD

**Issue by ghajini** [uBlock-user/uBO-Scriptlets#13]
Do you plan to fix `https://github.com/uBlockOrigin/uAssets/issues/5890` with your arsenal scriptlets

Link to test this `http://ouo.press/ANvl1r` 
you will get those doamains containing allow notifications

**Comment by uBlock-user** [comment]
I don't see any issue, I do get lots of redirections to crappy domains but I do reach the destination without having to allow any Notification on Firefox as said before in https://github.com/uBlockOrigin/uAssets/issues/5890#issuecomment-593863536

**Comment by uBlock-user** [comment]
Is this on `http://ouo.press/ANvl1r` ?

**Comment by ghajini** [comment]
they are bunch of domains ,i just provided original domain from where this start

**Comment by uBlock-user** [comment]
I can't test that url because of some IP limitation on some third-party crapware domain, but I can't reproduce when I tested on `https://ouo.press/8Ywl7`

**Comment by ghajini** [comment]
ok if ip limtation
 just go here `jetleech.net` 
insert any link eg `https://www111.zippyshare.com/v/4shWI3Pl/file.html` then following all proc
you get to that bypass requirement domain

**Comment by ghajini** [comment]
this is how it work `https://github.com/Sainan/Universal-Bypass/issues/468` ,i can't use userscript in firefox a due to unavailability of userscript manager

**Comment by uBlock-user** [comment]
> jetleech.net

I never reach the notifications page, i get stuck at please wait on `oui.today`...

tried again now i get to a completely different domain. If you can consistently repro, open a new issue or post in that old thread.

**Comment by ghajini** [comment]
> 
> 
> > jetleech.net
> 
> I never reach the notifications page, i get stuck at please wait on `oui.today`...
> 
you will stuck ,you have to click allow button then push notifications start ,you will redirected to desired location but at the expense of browser sending push notifications at browser interface & sometimes in background like this
![Screenshot (255)](https://user-images.githubusercontent.com/20338483/112337922-5fa4cc80-8ce4-11eb-8809-0a3ae5386d43.png)

**Comment by uBlock-user** [comment]
There's no allow button or anything, just a text that says `Please Wait....` I never reach the page in `https://github.com/uBlock-user/uBO-Scriptlets/issues/13#issuecomment-805899924`

**Comment by uBlock-user** [comment]
Tried again and I didn't get get stuck at that page at https://github.com/uBlock-user/uBO-Scriptlets/issues/13#issuecomment-805899924, reached the destination i.e at `https://exe.app/nXba` clicked getlink button and the Firefox download popups appears

![Capture](https://user-images.githubusercontent.com/21290713/112340037-31c08780-8ce6-11eb-9685-56e0187c5d02.PNG)

**Comment by ghajini** [comment]
i recorded
`i.imgur.com/oXeFtSW.gif`

**Comment by uBlock-user** [comment]
> 1st clear cookies+data

of which site ?

**Comment by ghajini** [comment]
above all sites or crapy ouo.today or test on fresh profile if you don't want to wholly delete cache+cookie

**Comment by uBlock-user** [comment]
As per your gif, `http://sprysphere.com/BiZZ`        -->     `https://www.dropbox.com/s/lwph95lsyuv21kw/LDOE_v1.8_Mod_iOSGods_Div.zip?dl=0`

**Comment by ghajini** [comment]
you get redirected but iam not &that is all issue all about `https://github.com/uBlockOrigin/uAssets/issues/5890`
hope i see userscript manager soon in firefox a

**Comment by uBlock-user** [comment]
> above all sites or crapy ouo.today or test on fresh profile if you don't want to wholly delete cache+cookie

tested on Firefox 84, with the instructions you posted in https://github.com/uBlock-user/uBO-Scriptlets/issues/13#issuecomment-805914922

Can't reproduce I get redirected to `https://tii.ai/9ebD` instead of getting stuck on Allow page and then it continues to `exe.io` and finally the destination `https://exe.app/nXba` as per https://github.com/uBlock-user/uBO-Scriptlets/issues/13#issuecomment-805937851

**Comment by uBlock-user** [comment]
Things you can do to see if it helps(based on the gif) --

1) Disable all extensions other than uBO

2) Browse to `about:preferences#privacy`  disable ETP if you're on std or strict, else if you're on custom, untick _Tracking content_
